### PR TITLE
[ADD] Provide code for chapter 9

### PR DESCRIPTION
**Pull request by PhilippKuhlmay**
Inserted code for chapter 9. 
Its not ready to merge.

### CODE REVIEW

**Comment by PhilippKuhlmay** [comment]
We should consider to have subfolders for the different steps inside the book. The code is updated in further steps so it should be usefull if we have something like "Your code should look like this after Step 1". And then all the updates for step 2, 3, 4, 5, ... are made in another folder.

**Comment by PhilippKuhlmay** [comment]
This is ready for review

**Review by t3jh — APPROVED** [review]
👍

File: Ch09/Step_4/promotionevents/Resources/Private/Templates/Events/Upcoming.html [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<h2>Upcoming Events</h2>
+<ul>
+   <li>Event a</li>
+   <li>Event b</li>
+   <li>Event c</li>
+</ul>
```
[the remark this answers is not in the document]
Comment (PhilippKuhlmay) [reply Ch09/Step_4/promotionevents/Resources/Private/Templates/Events/Upcoming.html]: Fixed it

File: Ch09/Step_6/promotionevents/Resources/Private/Templates/Events/Upcoming.html [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<h2>Upcoming Events</h2>
+<ul>
+   <f:for each="{upcomingEvents}" as="event">
+      <li>
+         {event.title} at {event.location} on <f:format.date format="d.m.Y">{event.promotiondate}</f:format.date><br>
+         <f:link.action action="detail" arguments="{event: event}">More details</f:link.action>
+      </li>
+   </f:for>
+</ul>
```
[the remark this answers is not in the document]
Comment (PhilippKuhlmay) [reply Ch09/Step_6/promotionevents/Resources/Private/Templates/Events/Upcoming.html]: Fixed it

**Review by tmfrstn — APPROVED** [review]

**Pull request merged**